microsoft-edge/devtools-guide-chromium/layers/layers-tool.md

```diff
@@ -1,23 +0,0 @@
----
-title: Layers tool
-description: In Microsoft Edge DevTools, the Layers tool represents the webpage in a 3D space.  The Layers tool has been replaced by the 3D View tool, and now directs to it.
-author: MSEdgeTeam
-ms.author: msedgedevrel
-ms.topic: conceptual
-ms.prod: microsoft-edge
-ms.technology: devtools
-ms.date: 03/04/2022
----
-# Layers tool
-
-Use the **3D View** tool instead; see [Navigate webpage layers, z-index, and DOM using the 3D View tool](../3d-view/index.md).
-
-The **Layers** tool (now the **3D View** tool) represents the webpage in a 3D space, to help you improve the rendering performance of your webpage.  Use the **Layers** tool to analyze how many layers are created by your CSS code, how big they are, and how often they change.  The **Layers** tool shows the different layers that are created by the browser rendering engine, and then displays a composited image of the webpage in a 3D space that you can rotate.
-
-The **Layers** tool is now superseded by the **Composited Layers** tab of the **3D View** tool.  The following screenshot shows the **Layers** tool open in **Quick View** using **Focus Mode**, instead of in the Drawer:
-
-![The Layers tool now links to the 3D View tool.](images/layers-link-3d-view-tool.png)
-
-The **Layers** tool previously showed the following:
-
-![The Layers tool.](../media/layers-tool.png)
```

Comment:
fulltext search repo for "layers-tool.png".  if no file uses this png, delete this png.

Reply:
Good point. No files use this, so I deleted it.